_posts/2021-11-05-r-6.0.0.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+title: Apache Arrow R 6.0.0 Release
+date: "2021-11-05"
+author: Nic Crane, Jonathan Keane, Neal Richardson
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+We are excited to announce the recent release of version 6.0.0 of the Arrow R package on CRAN. While we usually don't write a dedicated release blog post for the R package, this one is special. There are a number of major new features in this version, some of which we've been building up to for several years.
+
+# More dplyr support
+
+In version 0.16.0 (February 2020), we released the first version of the Dataset feature, which allowed you to query multi-file datasets using `dplyr::select()` and `filter()`. These tools allowed you to find a slice of data in a large dataset that may not fit into memory and pull it into R for further analysis. In version 4.0.0 earlier this year, we added support for `mutate()` and a number of other dplyr verbs, and all year we've been adding hundreds of functions you can use to transform and filter data in Datasets. However, to aggregate, you'd still need to pull the data into R.
+
+## Grouped aggregation
+
+With `arrow` 6.0.0, you can now `summarise()` on Arrow data, both with or without `group_by()`. These are supported both with in-memory Arrow tables as well as across partitioned datasets. Most common aggregation functions are supported: `n()`, `n_distinct()`, `min(),` `max()`, `sum()`, `mean()`, `var()`, `sd()`, `any()`, and `all()`. `median()` and `quantile()` with one probability are also supported and currently return approximate results using the t-digest algorithm.
+
+As usual, Arrow will read and process data in chunks and in parallel when possible to produce results much faster than one could by loading it all into memory then processing. This allows for operations that wouldn't fit into memory on a single machine.
+
```

Comment:
I think we might add a small example here. I saw an aggregation example below but I think it fits to have one here, as well.

Reply:
I know it breaks with the other nycflights examples, but we could do the nyc-taxi dataset here since it is bigger than memory and show an aggregation. Thoughts @jonkeane @thisisnic?

Reply:
This is great